index.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const helmet = require('helmet');
+
+const cors = (req, res, next) => {
+    res.header('Access-Control-Allow-Origin', '*');
+    res.header('Access-Control-Allow-Credentials', true);
+    res.header('Access-Control-Allow-Methods', 'PUT, GET, POST, DELETE, OPTIONS');
+    res.header('Access-Control-Allow-Headers', 'Content-Type');
+    next();
+}
+
+const db = require('./database/dbConfig.js')
+
+
+const server = express();
+const PORT = process.env.PORT || 4000;
+
+server.use(helmet());
+server.use(express.json());
+server.use(cors);
+
+
+server.post('/api/notes', (req, res)=>{
+    const note = req.body;
+    db.insert(note)
+    .then(ids =>{
+        db.findById(ids[0])
+        .then(notes=>{
+            res.status(201).json(note)
```

Comment:
Notice the difference between the callback's parameter name and the name of the variable you're sending back: `notes` vs `note`. The data that you're sending back is the `req.body` that the client sent over. This doesn't include the new id since you're not sending back your innermost callback parameter!